swagger.yml

```diff
@@ -2259,6 +2279,25 @@ definitions:
         type: string
       error:
         type: boolean
+  pluginRun:
+    type: object
+    properties:
```

Comment:
We should specify which properties are required.

Reply:
Skipping because this is an internal only endpoint that may change quickly, per our convo in Slack.